src/routes/_libraries/start.$version.index.tsx

```diff
@@ -84,13 +84,21 @@ export default function VersionIndex() {
         </p>
         <div className="flex justify-center gap-4 flex-wrap">
           <Link
-            to="./docs/framework/react/quick-start#impatient"
+            from={'/$libraryId/$version'}
+            to={'./docs/framework/$framework/$'}
+            params={{
+              libraryId: library.id,
+              framework: 'react',
+              _splat: 'quick-start#impatient',
```

Comment:
Instead of adding heading id into the splat using `#`, it'd probably better if the `hash` prop is used instead.

```
hash="impatient"
```